Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -145,6 +145,26 @@ ErrorOr<void> JPEGLoadingContext::decode()
                 }
             }
         }
+
+        // From libjpeg-turbo's libjpeg.txt:
+        // https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/libjpeg.txt
```

Comment:
This link gives me a 404

Reply:
This comment was copied from the original present in the previous JPEGLoader prior to libjpeg. I've replaced it :)